include/sound/sof/ipc4/header.h

```diff
@@ -536,6 +536,7 @@ struct sof_ipc4_notify_resource_data {
 #define SOF_IPC4_DEBUG_SLOT_DEBUG_LOG		0x474f4c00 /* byte 0: core ID */
 #define SOF_IPC4_DEBUG_SLOT_GDB_STUB		0x42444700
 #define SOF_IPC4_DEBUG_SLOT_TELEMETRY		0x4c455400
+#define SOF_IPC4_DEBUG_SLOT_TELEMETRY2		0x4c455500
```

Comment:
I'd assume this would require some discussion. The old telemetry slot is not very extendable so something new is needed to pass more debugging information to SOF side.